src/duct/module/web.clj

```diff
@@ -80,6 +82,8 @@
 
 (def ^:private api-config
   {:duct.handler.static/bad-request           {:body ^:displace {:error :bad-request}}
+   :duct.handler.static/bad-request-malformed {:body (cheshire/generate-string {:error "Malformed request body"})
```

Comment:
Generating JSON via Cheshire

Reply:
Wouldn't it be better to do this as part of the error handling? Otherwise `:duct.handler.static/bad-request-malformed` is treated differently from all the other handlers.